algos/custom_ppo2.py

```diff
@@ -4,7 +4,7 @@
 import gym
 import numpy as np
 from stable_baselines import logger, PPO2
-from stable_baselines.a2c.utils import total_episode_reward_logger
+from stable_baselines.common.tf_util import total_episode_reward_logger
 from stable_baselines.common import explained_variance, TensorboardWriter
 from stable_baselines.common.runners import AbstractEnvRunner
 from stable_baselines.ppo2.ppo2 import get_schedule_fn, safe_mean, swap_and_flatten
```

Comment:
some of those functions were moved too, no?
see https://stable-baselines.readthedocs.io/en/master/misc/changelog.html#id8

although it still works because ppo2 import them...

Reply:
yes sorry that wasn't very thorough, just changed the lines that were throwing an error when ran train.py .. I'll change the others too